Update logs_pipelines.md (#6637)
DataDog/documentation: content/en/api/logs_pipelines/logs_pipelines.md

## Logs Pipelines

> <mark>The Logs Pipelines endpoints are not supported in Datadog's client libraries. To request this functionality, contact [Datadog Support][1]. To permission your account see [Global Role Permissions][4]</mark>

Reviewer: ```suggestion
<mark>The Logs Pipelines endpoints are not supported in Datadog's client libraries. To request this functionality, contact [Datadog Support][1]. To set Pipeline permission see the [Granting permissions within limited scopes][4] documentation</mark>
```

[1]: /help
[2]: https://app.datadoghq.com/logs/pipelines

> [3]: https://docs.datadoghq.com/logs/processing

Reviewer: ```suggestion
[3]: /logs/processing
```

> [4]: https://docs.datadoghq.com/account_management/faq/managing-global-role-permissions/?tab=datadogussite#granting-permissions-within-limited-scopes

Reviewer: ```suggestion
[4]: /account_management/faq/managing-global-role-permissions/?tab=datadogussite#granting-permissions-within-limited-scopes
```
